cypress/support/constants/constants.js

```diff
@@ -325,6 +325,7 @@ module.exports = {
     RECORD_LIFECYCLE_HISTORY: 'fcs.recordLifecycleHistory',
     SETPERFORMANCETESTHANDLER: 'performance.setPerformanceTestHandler',
     SETLIFECYCLESTATE: 'fcs.setLifecycleState',
+    TRIGGEREVENT: 'fcs.triggerEvent',
```

Comment:
add this override function name to REQUEST_OVERRIDE_CALLS in constants

Reply:
This is added in REQUEST_OVERRIDE_CALLS only